doc/api/dns.md

```diff
@@ -629,6 +630,23 @@ array of host names.
 On error, `err` is an [`Error`][] object, where `err.code` is
 one of the [DNS error codes][].
 
+## `dns.setDefaultOrder(order)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `order` {string} must be `'ipv4'` or `'verbatim'`.
+
+Set the default value of `verbatim` in [`dns.lookup()`][] and
+[`dnsPromises.lookup()`][]. The value could be:
+* `ipv4`: sets default `verbatim` `false`.
```

Comment:
`ipv4first`? `ipv4` it not a particularly meaningful string when specifying the order of values.